.github/workflows/agent-stage.yml

```
@@ -1,20 +1,55 @@
 
 concurrency:
   group: ${{ github.ref }}-${{ github.workflow }}
   cancel-in-progress: true
 
 jobs:
+  check-release-tag:
+    # Guards the latest-tag race (see header comment): builds stamp the LATEST repo tag, so
+    # only build when this exact commit carries a release tag - i.e. 'stage-apps' was
+    # fast-forwarded to a 'stage' commit that 'Release Stage' has already tagged. Retries
+    # absorb the short merge -> tag delay. Ad-hoc pushes to 'temp' skip the check.
+    runs-on: ubicloud-standard-4
```

actionlint (1.7.12)

[error] label "ubicloud-standard-4" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2025-vs2026", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-intel", "macos-26-xlarge", "macos-26-large", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)